package.json

```diff
@@ -16,6 +16,7 @@
 		"licenses": "license-checker --production --csv --out licenses.csv"
 	},
 	"dependencies": {
+		"@react-hook/resize-observer": "^1.2.6",
```

Comment:
can we implement it without adding enw dependency? we already have resize-observer-polyfill and use it in project